src/middlewared/middlewared/alert/source/cores.py

```diff
@@ -14,12 +14,8 @@ class CoreFilesArePresentAlertClass(AlertClass):
             "Once the core files have been archived and attached to the ticket, they may be removed "
             "by running the following command in shell: 'rm /var/db/system/cores/*'.")
 
-    products = ("CORE", "SCALE")
-
 
 class CoreFilesArePresentAlertSource(ThreadedAlertSource):
-    products = ("CORE", "SCALE")
```

Comment:
@yocalebo was the intention here to disable this alert on ENTERPRISE systems?

Reply:
Yes, it was overwhelming support team.